cli/command/formatter/formatter.go

```diff
@@ -73,7 +73,7 @@ func (c *Context) preFormat() {
 func (c *Context) parseFormat() (*template.Template, error) {
 	tmpl, err := templates.Parse(c.finalFormat)
 	if err != nil {
-		return tmpl, errors.Errorf("Template parsing error: %v\n", err)
+		return tmpl, errors.Wrap(err, "template parsing error")
```

Comment:
FWIW; although the error changed (removing the newline at the end); when printed on the command-line, there's still a newline printed, so not sure why this was added in the first place 🤷‍♂️